enforcer/templates/enforcer-daemonset.yaml

```diff
@@ -60,8 +64,8 @@ spec:
       - name: enforcer
         image: "{{ .Values.global.imageCredentials.repositoryUriPrefix }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: "{{ .Values.image.pullPolicy }}"
-        securityContext:
-          {{- toYaml .Values.securityContext | nindent 10 }}
+        container_securityContext:
```

Comment:
Do you think there is a kubernetes pod variable named "container_securityContext"? Will kubernetes know how to assign security context to container by this variable? 
I didn't find any in their official documentation: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/